src/types-and-traits.rst

```diff
@@ -2749,3 +2758,67 @@ its lifetime elided form is
 
    fn f <T: ToCStr>(&mut self, args: &[T]) -> &mut Command;
 
+.. _fls_XTBOhK2Yk4lA:
+
+Trait Object Lifetime Elision
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. rubric:: Legality Rules
+
+:dp:`fls_URl9CeIVsiWs`
+The :t:`lifetime` of a :t:`trait object type` when :t:`elided` shall be inferred
+as follows:
```

Comment:
Similar to the comment on line 2811, a better phrasing might be

An elided lifetime of a trait object type is inferred as follows:

Also, I think this should be a fact because the programmer cannot perform inference.